package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "teston",
-  "version": "0.7.0",
+  "version": "0.0.0",
```

Comment:
Hmm.. why version `0.0.0`? 😄

Reply:
Sorry! I panicked. The plan from the beginning was not to push and I did not want to interfere with your versioning. I have pushed a new commit changing the version to 0.8.0.
I have some problem with GitHub. Do not know if my new commits is in this pull request.